### ISSUE TITLE
Badge position incorrect in codepen

### ISSUE THREAD

**Issue by xforceco** [google/material-design-lite#1341]
http://www.getmdl.io/components/index.html#badges-section

Wrong positioning when 'Edit on codepen'

**Comment by SeanBannister** [comment]
This looks like a duplicate of #708.
But thanks for the bug report.

**Comment by SeanBannister** [comment]
Just realised the badge on the text example is way off, which does look like a separate issue:
http://codepen.io/anon/pen/qdvXYb

**Comment by surma** [comment]
This should be fixed once we publish 1.0.3 (the microsite is always on master, which is why it is working there but not in the codepen). I’m closing this, but please reopen if the problem persists after 1.0.3.

**Comment by tleunen** [comment]
Could you use the latest stable version instead of master?

**Comment by surma** [comment]
@tleunen Yes, we have it on the road map to not have the microsite use a different version of MDL than the actual release. Just hasn’t been implemented yet.

**Comment by SeanBannister** [comment]
@surma Just re-checked master, unfortunately this bug is showing itself there.

**Comment by surma** [comment]
@SeanBannister Yes, because the codepens are pulling in MDL 1.0.2 from the CDN, but the fix will be in 1.0.3. Or am I misunderstanding you?

**Comment by SeanBannister** [comment]
@surma Sorry I should of explained better, I complied the JS and CSS replaced the versions in codepen so it was running master and the bug is still showing itself.

**Comment by surma** [comment]
Ah, okay, thanks for making sure. I’ll take a look.

**Comment by sgomes** [comment]
(because he said we would take a look)

**Comment by surma** [comment]
I’m an idiot! Sorry for all the back and forth, guys.

I thought this had been fixed on master because I merged my own PR prematurely. The PR has since been reverted and is due to be merged for v1.1 (because SemVer!).

So keep an eye on #1261. Once that is merged, it will be fixed :D

**Comment by SeanBannister** [comment]
@surma I haven't had a chance to test #1261 but I believe this issue is separate as I mentioned above. Check the [second example (codepen)](http://codepen.io/anon/pen/qdvXYb) in the [Badges documentation](http://www.getmdl.io/components/index.html#badges-section) the badge is **really** out of place.

**Comment by surma** [comment]
I should have looked closer at the codepen sooner. It is so far off to the right because the label is being applied to a `<div>` with `display: block`, which makes it span the entire width of the body. Change it to `display: inline-block` or use `<span>` and it will work. I will change that in the demo. 

Thanks for the hint, @SeanBannister.
